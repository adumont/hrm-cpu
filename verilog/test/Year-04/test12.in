33 09
51 f6
31 d9
34 8f
34 3e
25 5d
15 83
29 34
05 85
51 27
53 82
11 a8
09 bd
32 76
39 74
33 b7
33 d2
18 5f
59 3a
31 85
41 54
53 f4
02 1f
23 b7
52 bd
20 38
06 3d
07 c1
20 8d
12 72
58 17
39 f2
46 3b
50 34
33 5d
57 2d
44 6d
21 32
54 ad
