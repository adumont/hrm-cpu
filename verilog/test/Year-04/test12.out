f6
09
8f
d9
5d
3e
34
83
27
85
a8
82
76
bd
b7
74
5f
d2
85
3a
f4
54
b7
1f
38
bd
c1
3d
72
8d
f2
17
34
3b
2d
5d
32
6d
